### DISCUSSION TITLE
Newline extends comment to the next line

### DISCUSSION THREAD

**Discussion by nexovec** [helix-editor/helix#12349] [Q&A] [answered]
Hi, is there any way to disable this behavior where newline extends a comment to the next line? It's really annoying. Didn't use to do that some time before...

**Comment by kirawi** [comment, marked as answer]
`editor.continue-comments = false`

**Reply by nexovec** [reply]
That is not a valid option for helix `v24.7`.

**Reply by the-mikedavis** [reply]
Correct, this feature was merged after 24.07 (https://github.com/helix-editor/helix/pull/10996) and the config option was added a bit later (https://github.com/helix-editor/helix/pull/12213). If you are building from source you may need to update in order for the option to be available.

**Reply by TornaxO7** [reply]
Huh, but the continue-comment feature was only added in the master branch.
